packages/cli/src/util/postHog/__test__/postHog.test.ts

```diff
@@ -0,0 +1,108 @@
+import { capture } from "..";
+import * as postHogClient from "../client";
+import * as moduleManifestUtil from "../../moduleManifestUtil";
+import * as config from "../../config";
+
+describe("postHog", () => {
+  beforeEach(() => {
+    // jest.resetAllMocks();
+  });
+
+  afterEach(() => {
+    delete process.env.MM_DEV;
+  });
+
+  it("should call capture on the postHog client", () => {
+    const mockPostHogClient = { capture: jest.fn() };
+
+    jest
+      .spyOn(postHogClient, "postHogClient")
+      // @ts-ignore
```

Comment:
@psugihara if you have a sec, lmk if you have a better idea how to appease ts here

Reply:
fixed in a different type unsafe way, but seems better 🤷